src/app/core/services/issue.service.ts

```diff
@@ -199,11 +196,8 @@ export class IssueService {
 
   createTutorResponse(issue: Issue, response: string): Observable<Issue> {
     return forkJoin([this.githubService.createIssueComment(issue.id, response), this.updateIssue(issue)]).pipe(
-      map((responses) => {
-        const [githubComment, issue] = responses;
-        issue.updateDispute(githubComment);
-        return issue;
-      })
+      // eslint-disable-next-line arrow-body-style
+      map((responses) => (responses[1].updateDispute(responses[0]), responses[1]))
```

Comment:
ditto here. the logic seems incorrect

we are returning a tuple instead of `issue`